### ISSUE TITLE
HyperBert CLS/3 Fine-tuning configuration

### ISSUE THREAD

**Issue by gonconist** [semantic-web-company/wic-tsv#5]
Hi,

I used the code you provided ([HyperBert3.py](https://github.com/semantic-web-company/wic-tsv/blob/master/HyperBert/HyperBert3.py), [HyperBertCLS.py](https://github.com/semantic-web-company/wic-tsv/blob/master/HyperBert/HyperBertCLS.py)) to fine-tune both BERT-base and large (4 models in total) on WiC-TSV. 

Although for HyperBertCLS (base) the results I get on the dev set seem OK, this is not the case with the other 3 models. I specifically noticed that after a few steps of training all metrics converge to the same numbers (especially on bert-large).

Could you please provide me with the exact hyperparameters you used for fine-tuning? 

Here are the results I got on the dev set:

![0001](https://user-images.githubusercontent.com/32281275/121033973-51c7f900-c7b5-11eb-8006-9e02e6339fba.jpg)

**Comment by artreven** [comment]
Hi @gonconist ,

Thanks for reporting!
With HyperBert3_{base} the issue is actually the incorrect accuracy calculations, I have just pushed a new commit 15b118f80535cccab67b8859bd9fa1c9c00d3daf where accuracy is fixed. You can also see this from the F_1 score. Hope this helps!

These published models are not exactly the baseline models, though quite similar. I am not sure what happens with Bert large, I will try to investigate in the near future.
